e2e-tests/cypress/integration/common/as-1577.js

```diff
@@ -0,0 +1,89 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Given('an appellant has prepared an appeal for as-1577', () => {
```

Comment:
Just a question Frank, I've not seen such a long statement, is this simply checking a full application process?